lxd/api_cluster.go

```diff
@@ -2233,3 +2241,439 @@ func internalClusterRaftNodeDelete(d *Daemon, r *http.Request) response.Response
 
 	return response.SyncResponse(true, nil)
 }
+
+// swagger:operation POST /1.0/cluster/members/{name}/state cluster cluster_member_state_post
+//
+// Evacuate or restore a cluster member
+//
+// Evacuates or restores a cluster member.
+//
+// ---
+// consumes:
+//   - application/json
+// produces:
+//   - application/json
+// parameters:
+//   - in: body
+//     name: cluster
+//     description: Cluster member state
+//     required: true
+//     schema:
+//       $ref: "#/definitions/ClusterMemberStatePost"
+// responses:
+//   "200":
+//     $ref: "#/responses/Operation"
+//   "400":
+//     $ref: "#/responses/BadRequest"
+//   "403":
+//     $ref: "#/responses/Forbidden"
+//   "500":
+//     $ref: "#/responses/InternalServerError"
+func clusterNodeStatePost(d *Daemon, r *http.Request) response.Response {
+	name := mux.Vars(r)["name"]
+
+	var localNode string
+	var err error
+
+	err = d.cluster.Transaction(func(tx *db.ClusterTx) error {
+		localNode, err = tx.GetLocalNodeName()
+		if err != nil {
+			return err
+		}
+
+		return nil
+	})
+	if err != nil {
+		return response.SmartError(err)
+	}
+
+	// Forward request
+	if localNode != name {
+		resp := forwardedResponseToNode(d, r, name)
+		if resp != nil {
+			return resp
+		}
+	}
```

Comment:
Based on what I read about forwardedResponseToNode and ResolveTarget, I don't think you need the if statement.
If the target node is the local node, it will return nil, if it's not, it will forward and return a response.